.github/actions/build-sign-publish-chainlink/action.yml

```diff
@@ -183,6 +183,12 @@ runs:
         IMAGE_DIGEST=${{ fromJSON(steps.buildpush-root.outputs.metadata)['containerimage.digest'] }}
         echo "root_image_name=${IMAGE_NAME}" >> $GITHUB_ENV
         echo "root_image_digest=${IMAGE_DIGEST}" >> $GITHUB_ENV
+        echo '## Container Images' >> $GITHUB_STEP_SUMMARY
+        echo '### Root Image' >> $GITHUB_STEP_SUMMARY
+        echo "Image Name: ${IMAGE_NAME}"  >> $GITHUB_STEP_SUMMARY
+        echo "Image Digest: ${IMAGE_DIGEST}"  >> $GITHUB_STEP_SUMMARY
```

Comment:
We want to avoid printing out root details in summary as this (root image) shouldn't be consumed publicly or by internal users.